Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1670,6 +1670,20 @@ def lhegensim(fragment,howMuch):
     if k2 in PUDataSets:
         upgradeStepDict['RecoFullPUHGCAL'][k]=merge([PUDataSets[k2],{'-s':'RAW2DIGI,L1Reco,RECO'},upgradeStepDict['RecoFullHGCAL'][k]])
 
+    upgradeStepDict['HARVESTFullTracking'][k]={'-s':'HARVESTING:@trackingOnlyValidation',
```

Comment:
As I wrote below, in principle this should be `'HARVESTING:@trackingOnlyValidation+@trackingOnlyDQM'` to match the definition of "RecoFullTracking". Not a big deal though, especially if we do some cleanup/unification with #14540 afterwards.